### DISCUSSION TITLE
No analog of `assert forall by` for `forall_arith`; deprecate `forall_arith`?

### DISCUSSION THREAD

**Discussion by jaylorch** [verus-lang/verus#601] [Language design]
It would be nice if one could do `assert forall_arith |x| <some arithmetic expression> by { <proof> }` to establish `forall_arith |x| <some arithmetic expression>`, analogously to how one can do `assert forall |x| <some normal expression> by { <proof> }`.

**Comment by Chris-Hawblitzel** [comment]
Maybe the thing to do is to combine `forall` and `forall_arith` into a uniform `forall`, and just restrict the uses of the quantified variables to avoid boxing/triggering issues.

**Comment by utaal** [comment]
Addressed by https://github.com/verus-lang/verus/pull/625.

Shall we deprecate `forall_arith` once https://github.com/verus-lang/verus/pull/625 is merged?

**Comment by utaal** [comment]
`forall_arith` has been removed in https://github.com/verus-lang/verus/commit/eb54500367ecddd9f03b3233152073af261f5886.
